test/selenium/src/__tests__/visual-regression/create.test.ts

```diff
@@ -25,29 +25,45 @@ beforeEach( async () => {
 })
 
 afterAll( async () => {
-    await ctx.dispose()
+    if (ctx)
+        await ctx.dispose()
 })
 
-afterEach( async () => {
-    await ctx.screenSnap('final')
-})
+// afterEach( async () => {
+//     await ctx.screenSnap('final')
+// })
 
 beforeAll(async () => {
     await loginPage.login('admin', 'admin')
 })
 
 describe('project', () => {
-    it('has not visually regressed', async () => {
+    // it('has not visually regressed', async () => {
+    //     await projectCreate.get()
+    //     const img = Buffer.from(await projectCreate.screenshot(true), 'base64')
+    //     expect(img).toMatchImageSnapshot({customSnapshotsDir: '__image_snapshots__', customDiffConfig: {threshold: 0.01}})
```

Comment:
Another way to disable these is through the `NOOP` implementation in `test/selenium.ts`. Currently it `NOOPS` them if not running in headless mode, however that could be extended.